Assets/Scripts/Gameplay/GameplayObjects/Breakable.cs

```diff
@@ -0,0 +1,206 @@
+using System;
```

Comment:
I don't wanna spend time re-reviewing the whole thing, besides changing access to things, what other changes should I see here?

Reply:
Networked data, file and class name, merger of client and server bits and changes in OnNetworkSpawn and OnNetworkDespawn, that now execute bits of logic based on IsServer and IsClient. This style of change has happened in multiple classes, not just Breakable.